indigo/__init__.py

```diff
@@ -135,3 +137,11 @@
     "/documents",
     "/sources",
 ]
+
+TYPE_JOINING_UP_INITIATIVE_ALWAYS_FILTER_KEYS_LIST = [
+    "/status",
+]
+
+TYPE_JOINING_UP_INITIATIVE_FILTER_LISTS_LIST = [
+    "/documents",
```

Comment:
Pretty sure this is wrong? This should be an empty list I think.

By putting /documents in here, you are saying each document has it's own status flag so the record as a whole can be PUBLIC but some documents can be PRIVATE. What actually happens is that as there is no status flag on documents, info in the documents array is never made public at all.

If that's the intention, it should be in TYPE_JOINING_UP_INITIATIVE_ALWAYS_FILTER_KEYS_LIST

Reply:
Hi @odscjames thanks for this, I think I copied the way documents were handled from one of your other schemas. I think I then decided that documents didn't need a status and removed it from the schema. So I need to have another look at this and make sure the documents array/links are actually transfered over the API

Reply:
I've now removed the `TYPE_JOINING_UP_INITIATIVE_FILTER_LISTS_LIST`